minecraft_servers/craftergang/manifest.json

```diff
@@ -1,101 +1,97 @@
 {
-    "server_name": "craftergang",
-    "nice_name": "Craftergang",
-    "direct_ip": "craftergang.de",
-    "server_wildcards": [
-        "craftergang.net"
-    ],
-    "supported_languages": [
-        "de"
-    ],
-    "social": {
-        "discord": "https://discord.gg/ThCt8Pq",
-        "teamspeak": "craftergang.de",
-        "tiktok": "craftergang",
-        "youtube": "https://www.youtube.com/@craftergang",
-        "instagram": "craftergang"
+  "server_name": "craftergang",
+  "nice_name": "Craftergang",
+  "direct_ip": "craftergang.de",
+  "server_wildcards": [
+    "%.craftergang.net",

```

Comment:
Not a real wildcard
<img width="835" alt="Google Chrome 2024-08-09 14 58 36" src="https://github.com/user-attachments/assets/a5058b64-e832-464e-a6d3-d023ae34a6c9">